src/backend/src/modules/configuration/configuration.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import * as fs from 'fs';
+@Injectable()
+export class ConfigurationService {
+  getFeatures() {
+    try {
+      // TODO need to check how to get the path of config file
+      // hardcode in here or get from a env variables
+      const filePath = process.env.FEATURE_FLAG_FILE_PATH;
+      const fileName = process.env.FEATURE_FLAG_FILE_NAME;
+      const text = fs.readFileSync(filePath + fileName, 'utf8');
+      const jsonData = JSON.parse(text);
+
+      return jsonData;
+    } catch (error) {
+      console.log(`Read Feature Flag File ERROR: ${error}`);
```

Comment:
catching the error here and not throwing an error will result in API returning a 200 to the client, assuming the data is fetched correctly. I think this a 500 error that client should receive. Thoughts?

Also, you can keep the log in addition to throwing, but change it to console.error instead